data.sql

```diff
@@ -10,3 +10,18 @@ VALUES
   ('Gabumon', '2018-11-15', 2, true, 8),
   ('Pikachu', '2021-01-07', 1, false, 15.04),
   ('Devimon', '2017-05-12', 5, true, 11);
+
+
+/* Add species column to the animals table */
+ALTER TABLE ANIMALS ADD SPECIES VARCHAR(50);
+
+/* Insertion 2 */
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES 
+  ('Charmander', '2020-02-08', 0, false, 11, null),
+  ('Plantmon', '2021-11-15', 2, true, 5.7, null),
+  ('Squirtle', '1993-04-02', 3, false, 12.13, null),
+  ('Angemon', '2005-06-12', 1, true, 45, null),
+  ('Boarmon', '2005-06-07', 7, true, 20.4, null),
+  ('Blossom', '1998-10-13', 3, true, 17, null),
```

Comment:
* Kindly adjust the weights as some of the required weights are negative numbers kindly check the screenshot below 👇 
![image](https://github.com/diarisdiakite/vet-clinic/assets/52497246/5d878189-bc07-4bb7-9524-ccc24bdf9fe4)

Reply:
Thank you @Davidosky007, changes have been made.

Reply:
![image](https://github.com/diarisdiakite/vet-clinic/assets/36234985/dbe44471-6443-42fc-ba15-4219af809f52)

Reply:
and in the queries file as well
![image](https://github.com/diarisdiakite/vet-clinic/assets/36234985/083fba72-4d1d-4f35-8e7c-82bcd6d430dd)